### ISSUE TITLE
Change query on the invoice detail page.

### ISSUE THREAD

**Issue by davidgatti** [0x4447-bootstrap/0x4447_bootstrap_webpage_dashboard#318]
On the detail page of the invoice, you are showing the price of the invoice from the last invoice I think, but you should get that information from the same place you getting it when you list all the invoices of the year. Meaning from the main Invoice Item, and not from the invoice Item with extra details.

**Comment by acidbiscuit** [comment]
@davidgatti Invoice details query to `money` with sk `invoice#{invoiceId}` returns only timeline items and information about the invoice itself.

![dashboard_invoice_details](https://user-images.githubusercontent.com/11949520/131207132-ae3c29ee-429b-46bd-b50f-bf5265e9d1e1.png)

**Comment by davidgatti** [comment]
Hmm let see, I made this issue, because what happened, was that I did subscribe to a yearly plan, then switched to a monthly one, and then back to yearly I think. and on the table view, you were displaying $1, but on the detailed page you displayed $5. 

When I checked in the DDB, the general invoice Item, was set to $1 correctly, but the event invoices, had $5 because the invoices was for $5 but the charge was actually $1 due to extra money in the user account.

Knowing this, you still think you are getting the $ value on the detail page correctly?

**Comment by acidbiscuit** [comment]
@davidgatti Currently we take amount from the first event in the list (first index in the list), assuming that amount is the same for every event. So following question arise:
- Is it possible to have different amounts in the list of events? If yes, which amount should be displayed for the invoice?

**Comment by davidgatti** [comment]
Yes, there can be different amounts, you should display the amount from the summary Invoice item, the one with no event ID, the one that you use in the table view.

**Comment by davidgatti** [comment]
The reason for the difference is that, the invoice can be for $5, but if in your account you still have let say $4, then you will really be charged $1, the amount that is missing. Because you were already charged in the paste.

**Comment by acidbiscuit** [comment]
@davidgatti Got it. As you can see on the screenshot above, every item in `invoice#{invoiceId}` response has `event_id` so it is unclear which invoice should be displayed when visiting invoice page via URL.

**Comment by davidgatti** [comment]
ok, can you point me to the line where you query to display the table of invoices?

**Comment by acidbiscuit** [comment]
@davidgatti https://github.com/0x4447-bootstrap/0x_dashboard_webpage/blob/master/store/invoices.js#L56

**Comment by davidgatti** [comment]
Ok, not Good. I hope by the end of the year be able to Switch to GraphQL so I can make the queries for you, and so you don't have to think about this stuff anymore, but for the time being this is the fastest route.

You have to query for this SK: `invoice#summary#2021#08#in_1JTJqWG1wg42NuVoXs54s5sG` - this is the full example SK. And this are the options:

- To generate the table view: begins with `invoice#summary#2021`
- To get the actual details of the clicked invoice: sk equals `invoice#summary#2021#08#in_1JTJqWG1wg42NuVoXs54s5sG`

This type of SK `invoice#in_1JTK2wG1wg42NuVoxltLQNvl#evt_1JTK2yG1wg42NuVo9g96m6Qf` is just to show the progression of the invoice state through time.
